contracts/UFragmentsPolicy.sol

```diff
@@ -172,6 +170,22 @@ contract UFragmentsPolicy is Ownable {
         orchestrator = orchestrator_;
     }
 
+    function setRebaseFunctionGrowth(int256 rebaseFunctionGrowth_)
+        external
+        onlyOwner
+    {
+        require(rebaseFunctionGrowth_ >= 0);
+        rebaseFunctionGrowth = rebaseFunctionGrowth_;
+    }
+
+    function setRebaseFunctionUpperPercentage(int256 rebaseFunctionUpperPercentage_)
```

Comment:
We should add a setter for the lower now, too.

Reply:
Done.